CounterStrike/src/dev/danablend/counterstrike/CounterStrike.java

```diff
@@ -822,12 +884,17 @@ public GameState getGameState() {
     }
 
 
+    public void setGameState(GameState newGameState) {
+         gameState = newGameState;
+    }
+
+
     public boolean isTNTDropped() {
         Location loc = this.getTerroristSpawn(false);
 
         for (Entity ent : loc.getWorld().getEntities()) {
             if (!(ent instanceof Player) && (ent.getType().equals(Material.TNT) || ent.getName().equals("TNT"))) {

```

Comment:
Often Java programmers use a little trick here. Instead of using myVariable.equals("TNT"), it may be used "TNT".equals(myVariable). Advantage of this solution is when myVariable is null. With the order you have chosen (which is obviously natural to choose when writing code), you may encounter NullpointerExceptions. If the order would be reversed, it would just compare TNT.equals(null), so it would return false without an exception.